pkg/controller/traverse.go

```diff
@@ -0,0 +1,37 @@
+package controller
+
+import (
+	"github.com/golang/glog"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+	"github.com/kubernetes-incubator/service-catalog/pkg/controller/storage"
+)
+
+func brokerForServiceClass(storage storage.Storage, cls *servicecatalog.ServiceClass) (*servicecatalog.Broker, error) {
+	// Get the broker for the serviceclass.
+	broker, err := storage.Brokers().Get(cls.BrokerName)
```

Comment:
TODO: make sure that the return param behavior of the mem storage and kubernetes storage is consistent